src/main/java/org/broadinstitute/consent/http/models/Draft.java

```diff
@@ -0,0 +1,142 @@
+package org.broadinstitute.consent.http.models;
+
+import com.google.gson.Gson;
+import java.util.Date;
+import java.util.HashSet;
+import java.util.Optional;
+import java.util.Set;
+import java.util.UUID;
+import org.broadinstitute.consent.http.models.dataset_registration_v1.DatasetRegistrationSchemaV1;
+
+/**
+ * DraftSubmission represents a Data Submitter's partial submission of 0 or more elements. This is
+ * an internal structure, not intended to be serialized back to the client in the entirety.
+ */
+public class Draft implements DraftInterface {
+
+  private Date createDate;
+  private Date updateDate;
+  private Set<FileStorageObject> storedFiles;
+  private transient String json;
```

Comment:
Is there another way in gson to declare a field as non-serialized? This use will flag a warning in sonar since it only expects `transient` inside a `Serializable` class.

Reply:
From the [GSON manual](https://google.github.io/gson/UserGuide.html#excluding-fields-from-serialization-and-deserialization), there are three supported ways to do it:
- If a field is marked transient, (by default) it is ignored and not included in the JSON serialization or deserialization.
- We can decorate all of the properties we want to @Expose and use a different pattern on drafts, users, filestorage and other classses we make use of.
- We can add a filter, add field decorators, and apply the filter  at the point of serialization.

If you think the second or third option is worth the squeeze, I'll implement.